test/bot.test.ts

```diff
@@ -0,0 +1,5 @@
+import { sum } from '../src/bot-lib'
```

Comment:
@Velenir for some reason, it does't like `import { sum } from '../src/bot-lib'` 

I added `"baseUrl": "src"` in `tsconfig.json`, but apparently, it still don't like it!
Any idea?

Reply:
i think it's because it's outside the baseUrl

Reply:
How do you run it? I don't see `ts-node` in scripts and I expected something like
```sh
nodemon --exec ts-node src/bot.ts
```

Reply:
For jest:
https://github.com/gnosis/dex-telegram/pull/11/files#diff-2d0cd5d10b9604941c38c6aac608178aR90

For dev: nodemon is smart enough to see is TS and use ts-node:
https://github.com/gnosis/dex-telegram/pull/11/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R8

![image](https://user-images.githubusercontent.com/2352112/68487683-cb97a980-0243-11ea-95a9-cd5781be4eb6.png)


For production, you build and then run the transpiled files

Reply:
> i think it's because it's outside the baseUrl

Not sure. In `dex-react` I think I did something different, cause it's working there, but I just don't see the difference now...

![image](https://user-images.githubusercontent.com/2352112/68487911-4660c480-0244-11ea-904f-8d2df4998ba7.png)

Reply:
removing the `include` fixes this @anxolin

Reply:
Nice! Thanks, that was it!

Reply:
Ups! it worked momentarily :(

And travis don't like it anymore. We can take a look on monday to this.
https://travis-ci.org/gnosis/dex-telegram/builds/609360379?utm_source=github_status&utm_medium=notification

Reply:
It seems there's several typescript problems, I'll create an issue just for that and apply a workaround so I can merge this PRs that are base for next ones: 
* https://github.com/gnosis/dex-telegram/issues/14